modules/stateful_config/versions.tf

```diff
@@ -15,7 +15,7 @@
  */
 
 terraform {
-  required_version = "~> 0.12.6"
+  required_version = ">= 0.12.6"
   required_providers {
     google = ">= 2.7, <4.0"
   }
```

Comment:
I think we should remove the provider version here, we control this via our providers config
like here
https://github.com/recurly/recurly-gcp/blob/main/non-production/main_providers.tf#L14-L18

Reply:
good catch!